src/fights.ts

```diff
@@ -404,12 +408,24 @@ function embezzlerSetup() {
 
   bathroomFinance(embezzlerCount());
 
+  const averageEmbezzlerNet = ((baseMeat + 750) * meatDropModifier()) / 100;
+  const averageTouristNet = (baseMeat * meatDropModifier()) / 100;
+
   if (SourceTerminal.have()) SourceTerminal.educate([$skill`Extract`, $skill`Digitize`]);
-  if (!get("_cameraUsed") && !have($item`shaking 4-d camera`)) {
+  if (
+    !get("_cameraUsed") &&
+    !have($item`shaking 4-d camera`) &&
+    averageEmbezzlerNet - averageTouristNet > mallPrice($item`4-d camera`)
+  ) {
     retrieveItem($item`4-d camera`);
   }
 
-  if (!get("_iceSculptureUsed") && !have($item`ice sculpture`)) {
+  if (
+    !get("_iceSculptureUsed") &&
+    !have($item`ice sculpture`) &&
+    averageEmbezzlerNet - averageTouristNet >
+      mallPrice($item`snow berries`) + mallPrice($item`ice harvest`) * 3
```

Comment:
Is it worth also checking the direct price of a sculpture?

Reply:
it's not tradable